### PR TITLE
Add Fernet encryption/decryption operation

### PR DESCRIPTION
**Pull request by kassi**
This PR adds fernet encryption/decryption operation.

It uses fernet.js module to perform the actual de/encryption (MIT license).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -96,6 +96,7 @@
     "esmangle": "^1.0.1",
     "esprima": "^4.0.1",
     "exif-parser": "^0.1.12",
+    "fernet": "^0.3.1",
```
Comment (mattnotmitt) [inline package.json:99]: Update to new release of fernet, `0.3.2`

File: src/core/operations/FernetDecrypt.mjs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        ];
+    }
+    /**
+     * @param {String} input
+     * @param {Object[]} args
+     * @returns {String}
+     */
+    run(input, args) {
+        const [secretInput] = args;
+        // const fernet = require("fernet");
```
Comment (mattnotmitt) [inline src/core/operations/FernetDecrypt.mjs:49]: Remove dead code.

**Review by mattnotmitt — APPROVED** [review]
LGTM, some changes before merge @a3957273.

**Comment by a3957273** [comment]
Thanks so much for the contribution @kassi , apologies it took us so long to merge it in. :)

**Pull request merged**